index.html

```diff
@@ -70,25 +70,28 @@
 The <dfn>Server-Timing header field</dfn> is used to communicate one or more metrics and descriptions for the given request-response cycle. The ABNF (Augmented Backus-Naur Form) [[!RFC5234]] syntax for the <a>Server-Timing header field</a> is as follows:
 
 ```ABNF
-  Server-Timing        = #server-timing-metric
-  server-timing-metric = metric  [ OWS ";" OWS description ]
-  metric               = metric-name [ OWS "=" OWS metric-value ]
-  metric-name          = token
-  metric-value         = *DIGIT [ "." 1*DIGIT ]
-  description          = token | quoted-string
+  Server-Timing             = #server-timing-metric
+  server-timing-metric      = metric-name *( OWS ";" OWS server-timing-param )
+  metric-name               = token
+  server-timing-param       = server-timing-param-name OWS "=" OWS server-timing-param-value
+  server-timing-param-name  = token
+  server-timing-param-value = token / quoted-string
 ```
 
-* Each metric MUST have a name.
-  * Metric names MAY be repeated.
-  * Metric names are defined by the server.
-* Each metric MAY have an optional numeric value.
-* Each metric MAY have an optional description.
-* The user agent MAY surface provided metrics in any order - i.e. the order of metrics in the HTTP header field is not significant.
+See [[!RFC7230]] for definitions of `#`, `*`, `OWS`, `token`, and `quoted-string`.
 
-See [[!RFC7230]] for definitions of `#`, `OWS`, `token`, `DIGIT`, and `quoted-string`.
+On a response, there can be any number of server-timing-metric entries with the same metric-name.
```

Comment:
"A response MAY have multiple server-timing-metric entries with the same metric-name."

Also, we should spell out how the UA should behave when that's the case.. 

~", and the user agent SHOULD process all such entries with same metric-name." or some such.